docs_src/design/adr/0024-system-events.md

```diff
@@ -37,23 +37,24 @@ This new `SystemEvent` DTO will contain the following data describing the System
 
 Services that publish System Events (Core Metadata) must connect to the EdgeX MessageBus and have MessageBus configuration similar to that of Core Data's [here](https://github.com/edgexfoundry/edgex-go/blob/v2.2.0/cmd/core-data/res/configuration.toml#L53-L74). This design assumes that Core Metadata will have this capability and configuration due to planned implementation of Service Metrics. 
 
-For this design, the `publish topic prefix` for System Events will be a constant in the code rather than another configuration item that never changes.
+The `PublishTopicPrefix` property in Core Metadata's `MessageQueue` configuration will be used for System Events and set to `edgex/system-event`.
 
 #### MessageBus Topic
 
-The new `SystemEvent` DTO will be published to a multi-level topic allowing subscribers to filter by topic. The format of this topic for `Device System Events` will be:
+The new `SystemEvent` DTO will be published to a multi-level topic allowing subscribers to filter by topic. The format of this topic for **System Events** will be:
 
-​		`edgex/system-event/{source}/{type}/{action}/{owner}/{profile}`
+​		`edgex/system-event/{source}/{type}/{action}`
```

Comment:
Would it more properly be `{PublishTopicPrefix}/{source}/{type}/{action}` ?

Reply:
Sure.